apps/web/tsconfig.json

```diff
@@ -27,9 +25,23 @@
       "~/utils/*": ["src/utils/*"]
     },
     "outDir": "../../.moon/cache/types/apps/web",
-    "incremental": true
+    "incremental": true,
+    "noEmit": false
```

Comment:
Without explicitly setting `noEmit` to `false`, Next.js automatically overwrites this file with `noEmit: true` which is not what we want.